app/views/publisher_information/_publication_info_nav.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="govuk-grid-column-one-third">
+  <ul class="govuk-list">
+    <li>
+      <% if current_page == beta_capabilities_path %>
+        <strong>What the Beta can and cannot do</strong>
+      <% else %>
+        <a href=<%= beta_capabilities_path %> class="govuk-link govuk-link--no-visited-state">What the Beta can and cannot do</a>
```

Comment:
seem to have missed quotes off the href quotes here - no biggie

Reply:
FIXED.  I scanned anywhere else this might occur and it seems limited to this page.